build/ferretdb.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=FerretDB Database
+Documentation=https://docs.ferretdb.io/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/ferretdb
```

Comment:
Do we need a way to let users provide configuration of FerretDB? You commented `EnvironmentFile` is not recommended, but what about `Environment` string containing env vars? It is not recommended either?

Reply:
Let's add `Environment` options. They will also serve as an example for `systemctl edit ferretdb` command